main.go

```diff
@@ -1,9 +1,9 @@
-package main // import "github.com/UpCloudLtd/terraform-provider-upcloud"
+package main
 
 import (
 	"github.com/hashicorp/terraform/plugin"
 	"github.com/hashicorp/terraform/terraform"
-	"github.com/UpCloudLtd/terraform-provider-upcloud/upcloud"
+	"github.com/nikohaap/terraform-provider-upcloud/upcloud"
```

Comment:
Can you clarify the purpose of this change?

Reply:
It was jut because I was using the forked package instead of the UpCloudLtd. 
It should be the UpCloudLtd version instead of the forked (nikohaap) one :)